public/javascripts/components/nav.js

```diff
@@ -6,29 +6,111 @@ var navbar = Vue.extend({
           <h1 v-link="{path: '/'}" class="cc-logo">Chubbo-Chat</h1>
         </div>
         <div class="col-xs-4">
-          <p v-show="loginStatus" v-on:click='handleLogin' class="cc-loginBtn">login</p>
-          <p v-else v-on:click='handleLogout' class="cc-logoutBtn">logout</p>
+          <img
+            v-on:click='handleMenu'
+            src="/images/hamburger.svg"
+            class="cc-menuIcon"
+            id={{menuIconId}} 
+          />
+          <div id={{loginButtonsId}}>
+            <p
+              v-show="!user"
+              v-on:click='handleLogin'
+              class="cc-loginBtn"
+            >
+              login
+            </p>
+            <p
+              v-else
+              v-on:click='handleLogout'
+              class="cc-logoutBtn"
+            >
+              logout
+            </p>
+          </div>
+          <p v-link="{path: '/dashboard'}" id={{dashButtonId}}>
+            dashboard
+          </p>
         </div>
       </div>
     </div>
   `,
   data: function() {
-    return {loginStatus: firebase.auth().currentUser};
+    return {
+      user: firebase.auth().currentUser
+    };
+  },
+  ready: function() {
+    var me = this;
+    //set user to be based on store
+    window.setTimeout(function() {
+      me.user = me.userInStore.uid;
+    }, 1000);
+  },
+  computed: {
+    loginButtonsId: function() {
+      if (this.$route.path === '/') {
+        if (this.userInStore.uid) {
+          return 'cc-loginBtns-hide';
+        } else {
+          return 'cc-loginBtns-show';
+        }
+      } else if (this.$route.path === '/dashboard') {
+        return 'cc-loginBtns-hide-mobile';
+      }
+    },
+    menuIconId: function() {
```

Comment:
can we switch these two blocks of code to some kind of v-if ? I think it makes more sense to digest this logic in-place in the template, with the elements it effects